website/project/model.py

```diff
@@ -75,16 +74,9 @@ def has_anonymous_link(node, auth):
     :param str link: any view-only link in the current url
     :return bool anonymous: Whether the node is anonymous to the user or not
     """
-    view_only_link = auth.private_key or request.args.get('view_only', '').strip('/')
```

Comment:
Added in:

70f55a52e520a1ab2870562ba4c0ba4885dabbbe website/project/views/comment.py
22df9981d6cf3d3483ca6728a1dda4093c2e36cc website/project/views/comment.py
899daff8e0e625c86ac29d1fb5251ec195787a6b website/project/views/contributor.py
899daff8e0e625c86ac29d1fb5251ec195787a6b website/project/views/contributor.py
1854bb24045db2f75636d3d6d7ee76733ed00b20 website/project/views/node.py ← renamed!
1854bb24045db2f75636d3d6d7ee76733ed00b20 website/project/views/node.py